neuralpy/layers/bilinear.py

```diff
@@ -0,0 +1,107 @@
+"""Dense layer for NeuralPy"""
+
+from torch.nn import Bilinear as BiLinear
```

Comment:
Change `from torch.nn import Bilinear as BiLinear` to `from torch.nn import Bilinear as _BiLinear`